locals.tf

```diff
@@ -0,0 +1,138 @@
+locals {
+

```

Comment:
are we really using all these?

Reply:
I revised the locals.tf file to be as simple as possible